README.md

```diff
@@ -171,25 +106,37 @@ File: hosts
 injector ansible_host=myinjectorhost.addr.com ansible_user=centos
 ```
 
-Configure 3scale portal endpoint and services
+Configure the following settings in `roles/profiled-traffic-generator/defaults/main.yml` file:
+* `threescale_portal_endpoint`: 3scale portal endpoint 
+* `traffic_profile`: Currently available profiles: `simple, backend`
+* `private_base_url`: Private Base URL used for the tests
 
 ```
-File: roles/injector-configuratior/defaults/main.yml
+File: roles/profiled-traffic-generator/defaults/main.yml
 
 # URI that includes your password and portal endpoint in the following format: <schema>://<password>@<admin-portal-domain>.
 # The <password> can be either the provider key or an access token for the 3scale Account Management API.
 # <admin-portal-domain> is the URL used to log into the admin portal.
 # Example: https://access-token@account-admin.3scale.net
 threescale_portal_endpoint: <THREESCALE_PORTAL_ENDPOINT>
 
-# Comma separated list of services (Id's or system names)
-# If empty, all available services will be used
-threescale_services: ""
+# Used traffic for performance testing is not real traffic.
+# It is synthetically generated traffic based on traffic models.
+# Information about available traffic profiles (or test plans) can be found here:
+# TODO add link
```

Comment:
Is this TODO intentional? is there something missing?

Reply:
I forgot to fill it. Reminders are good thing